packages/now-go/index.ts

```diff
@@ -42,10 +43,15 @@ async function initPrivateGit(credentials: string) {
  * The output file is not renamed because v3 builders can't rename outputs
  * which works great for this feature.
  */
-async function getRenamedEntrypoint(entrypoint: string, files: Files) {
+async function getRenamedEntrypoint(
+  entrypoint: string,
+  files: Files,
+  meta: Meta
+) {
   const filename = basename(entrypoint);
   if (filename.startsWith('[')) {
-    const newEntrypoint = entrypoint.replace('/[', '/now-bracket[');
+    const suffix = meta.isDev && !entrypoint.endsWith('.go') ? '.go' : '';
```

Comment:
I would add a comment for both of these lines explaining _why_, cause it's not immediately clear without going back through the issue tracker.